### PR TITLE
Nisha | Test | Table header sticky

### PR DESCRIPTION
**Pull request by nisha-walkover**
https://app.clickup.com/t/86cwf7q3q

### CODE REVIEW

File: apps/web-giddh/src/assets/css/global/table.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
     .mat-mdc-header-row {
         .mat-mdc-header-cell {
             background-color: var(--color-body-background) !important;
+            z-index: 9;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/assets/css/global/table.scss]: Try once z-index 1

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 2]
```diff
@@ -2389,6 +2389,10 @@ ul.sub-list {
     width: 150px;
 }
 
+.w-fit-content {
+    width: fit-content
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/assets/css/style-2.scss]:
```suggestion
    width: fit-content !important;
```

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/css/global/table.scss (hunk 1 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/assets/css/global/table.scss]: done

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 2 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**